src/components/LottoGame/LottoTicketList/index.js

```diff
@@ -1,54 +1,43 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 
-class LottoTicketItem extends Component {
-  render() {
-    return (
-      <div className="lotto-wrapper flex items-center">
-        <span className="lotto mx-1 text-4xl">🎟️ </span>
-        {this.props.isToggleOn && <span className="ml-3 text-2xl ">{this.props.numbers.join(', ')}</span>}
-      </div>
-    );
-  }
-}
-
-export default class LottoTicketList extends Component {
-  constructor(props) {
-    super(props);
+const LottoTicketItem = ({ isToggleOn, numbers }) => {
+  return (
+    <div className="lotto-wrapper flex items-center">
+      <span className="lotto mx-1 text-4xl">🎟️ </span>
+      {isToggleOn && <span className="ml-3 text-2xl ">{numbers.join(', ')}</span>}
+    </div>
+  );
+};
 
-    this.state = {
-      isToggleOn: false,
-    };
-    this.handleToggle = this.handleToggle.bind(this);
-  }
+const LottoTicketList = ({ lottoTickets }) => {
+  const [isToggleOn, setToggle] = useState(false);
```

Comment:
```suggestion
  const [isOn, toggle] = useReducer(prev => !prev, false);
```

이렇게 해보면 어떨까요? (useState만 있는게 아니랍니다 :))

Reply:
useReducer는 하나의 state를 컨트롤하는 여러개의 액션이 있는 경우에만 사용한다고 생각했는데 이렇게도 사용할 수 있군요 👍👍👍